tools/sklearn/fitted_model_eval.xml

```diff
@@ -0,0 +1,75 @@
+<tool id="fitted_model_eval" name="Evaluate a Fitted Model" version="@VERSION@">
+    <description>using a new batch of labeled data</description>
+    <macros>
+        <import>main_macros.xml</import>
+        <import>keras_macros.xml</import>
+    </macros>
+    <expand macro="python_requirements"/>
+    <expand macro="macro_stdio"/>
+    <version_command>echo "@VERSION@"</version_command>
+    <command>
+        <![CDATA[
+        export HDF5_USE_FILE_LOCKING='FALSE';
+        python '$__tool_directory__/fitted_model_eval.py'
+            --inputs '$inputs'
```

Comment:
Why do you feed some inputs in via a config file and others not?

Reply:
Due to a Galaxy bug, the config file was unable to include input file paths. A good news is that the bug got fixed a few days ago.